src/test/java/uk/gov/digital/ho/proving/income/audit/statistics/PassStatisticsCalculatorTest.java

```diff
@@ -20,91 +22,95 @@
     private static final LocalDate TO_DATE = LocalDate.of(2019, Month.JANUARY, 31);
     private static final LocalDate IN_RANGE = FROM_DATE.plusDays(2);
 
+    private static final List<AuditResultByNino> SOME_AUDIT_RESULTS = emptyList();
+    private static final List<ArchivedResult> SOME_ARCHIVED_RESULTS = emptyList();
+    private static final List<String> SOME_CORRELATION_IDS = emptyList();
+
     private PassStatisticsCalculator accumulator = new PassStatisticsCalculator();
 
     @Test
     public void result_givenFromDate_inResult() {
-        List<AuditResultByNino> someList = emptyList();
-
         LocalDate fromDate = LocalDate.of(2019, 2, 3);
-        assertThat(accumulator.result(someList, fromDate, SOME_DATE).fromDate()).isEqualTo(fromDate);
+
+        assertThat(accumulator.result(SOME_AUDIT_RESULTS, SOME_ARCHIVED_RESULTS, fromDate, SOME_DATE).fromDate())
+            .isEqualTo(fromDate);
     }
 
     @Test
     public void result_givenToDate_inResult() {
-        List<AuditResultByNino> someList = emptyList();
-
         LocalDate toDate = LocalDate.of(2019, 3, 4);
-        assertThat(accumulator.result(someList, SOME_DATE, toDate).toDate()).isEqualTo(toDate);
+
+        assertThat(accumulator.result(SOME_AUDIT_RESULTS, SOME_ARCHIVED_RESULTS, SOME_DATE, toDate).toDate())
+            .isEqualTo(toDate);
     }
 
     @Test
     public void result_emptyList_emptyResult() {
-        assertThat(accumulator.result(emptyList(), FROM_DATE, TO_DATE))
+        assertThat(accumulator.result(emptyList(), emptyList(), FROM_DATE, TO_DATE))
```

Comment:
Are these empty lists the "SOME" constants (and maybe they should be "ANY" constants - see above)